staging/src/github.com/openshift/openshift-controller-manager/pkg/build/controller/build/build_controller.go

```diff
@@ -405,6 +460,7 @@ func (bc *BuildController) Run(workers int, stopCh <-chan struct{}) {
 
 	// Only need one worker to manage config
 	go wait.Until(bc.controllerConfigWorker, time.Second, stopCh)
+	go wait.Until(bc.proxyCfgWorker, time.Second, stopCh)
```

Comment:
Why the second worker to manage config?

Reply:
why not I guess ... my initial inclination was to fully separate the controller config processing from the proxy config processing 

In particular the controller config currently already piggy backs the `bc.readClusterImageConfig()` processing 

But I don't feel super strongly about it if you want me to consolidate through `bc.controllerConfigWorker`

Please confirm ... thanks

Reply:
Consolidating in one worker ensures that we are doing level-based reconciliation. Each time we get a watch event from our config CRDs, we sync the entire configuration state into the build controller's memory.

This is considered an operator best practice - see https://blog.openshift.com/kubernetes-operators-best-practices/ published

Reply:
ok I'll switch

Reply:
change to one worker for config pushed @adambkaplan ... PTAL